src/mmw/apps/modeling/models.py

```diff
@@ -79,7 +79,7 @@ class Meta:
     name = models.CharField(
         max_length=255)
     project = models.ForeignKey(Project, related_name='scenarios')
-    current_condition = models.BooleanField(
+    is_current_conditions = models.BooleanField(
```

Comment:
Can we replace this boolean with something more descriptive? For example, the name of the model simulation to run for this scenario? If it's null we can infer that the scenario is the "current condition".